packages/components/package.json

```diff
@@ -45,17 +45,18 @@
     "node": ">=16.0.0"
   },
   "dependencies": {
-    "@react-email/body": "0.0.2",
-    "@react-email/button": "0.0.9",
+    "@react-email/body": "0.0.3",
+    "@react-email/button": "0.0.10",
     "@react-email/column": "0.0.7",
     "@react-email/container": "0.0.8",
-    "@react-email/font": "0.0.2",
+    "@react-email/font": "0.0.3",
     "@react-email/head": "0.0.5",
     "@react-email/heading": "0.0.8",
     "@react-email/hr": "0.0.5",
-    "@react-email/html": "0.0.4",
+    "@react-email/html": "0.0.5",
     "@react-email/img": "0.0.5",
     "@react-email/link": "0.0.5",
+    "@react-email/markdown": "0.0.4",
```

Comment:
I believe all other packages ended up being updated already, but adding `markdown` can still be useful, so feel free to rebase to keep this change only 🙏

Reply:
Thanks for the update @vcapretz 

I've made the requested changes